Userland/Services/SystemServer/main.cpp

```diff
@@ -72,18 +72,19 @@ static ErrorOr<void> determine_system_mode()
             g_system_mode = "text";
     });
 
-    auto f = Core::DeprecatedFile::construct("/sys/kernel/constants/system_mode");
-    if (!f->open(Core::OpenMode::ReadOnly)) {
-        dbgln("Failed to read system_mode: {}", f->error_string());
+    auto f_or_error = Core::File::open("/sys/kernel/constants/system_mode"sv, Core::File::OpenMode::Read);
```

Comment:
Nit: I know it had the name `f` before, but can we call it `file_or_error` instead?